cmd/substreams/init.go

```diff
@@ -21,15 +28,36 @@ func init() {
 }
 
 func runSubstreamsInitE(cmd *cobra.Command, args []string) error {
-	projectName := args[0]
-	path := args[1]
+	srcDir, err := filepath.Abs("/tmp/test")
+	if err != nil {
+		return fmt.Errorf("getting absolute path of working directory: %w", err)
+	}
+	if len(args) == 1 {
+		srcDir, err = filepath.Abs(args[0])
+		if err != nil {
+			return fmt.Errorf("getting absolute path of given directory: %w", err)
+		}
+	}
 
-	srcDir, err := filepath.Abs(path)
+	// Bubble Tea model for questionnaire
+	questionnaireModel, err := tea.NewProgram(init_models.NewQuestionaire()).Run()
 	if err != nil {
-		return fmt.Errorf("getting absolute path of %q: %w", path, err)
+		return fmt.Errorf("creating questionnaire: %w", err)
+	}
+	questionnaireExposed := questionnaireModel.(init_models.Questionaire)
+	networkSelected := questionnaireExposed.Network.Selected
+	nameSelected := questionnaireExposed.ProjectName.TextInput.Value()
+
+	if networkSelected == "other" {
+		fmt.Println("We haven't added any templates for your selected chain quite yet...")
+		fmt.Println("Come join us in discord at https://discord.gg/u8amUbGBgF and suggest templates/chains you want to see!")
+		return nil
+	} else {
+		fmt.Printf("\033[32m ✔"+"\033[0m"+" Name: %s\n", nameSelected)
+		fmt.Printf("\033[32m ✔"+"\033[0m"+" Network: %s\n\n", networkSelected)
```

Comment:
There is good library for this to avoid coding the escape sequences. I used github.com/logrusorgru/aurora in the past, it's used in `logging` library to provide colors to logs.